src-example/index.js

```diff
@@ -2,21 +2,22 @@
 import 'react-hot-loader/patch'
 import React from 'react'
 import { render } from 'react-dom'
-import { createHistory } from 'history'
-import { Router, useRouterHistory } from 'react-router'
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
You are seeing an outdated commit (that came from `master`). See this one: https://github.com/diegohaz/arc/blob/23d4d2622c91bcbf5eb98bdb45d081860071fd7e/src-example/index.js